packages/jest-runtime/src/index.ts

```diff
@@ -83,10 +83,17 @@ type InitialModule = Partial<Module> &
 type ModuleRegistry = Map<string, InitialModule | Module>;
 type ResolveOptions = Parameters<typeof require.resolve>[1];
 
-type BooleanObject = Record<string, boolean>;
-type CacheFS = {[path: string]: string};
-
-type RequireCache = {[key: string]: Module};
+type StringMap = Map<string, string>;
+type BooleanMap = Map<string, boolean>;
+
+const fromEntries: typeof Object.fromEntries =
```

Comment:
`Object.fromEntries` was added in node 12.

Can ditch this when `jest-resolve` uses a `Map` as well